Mars-SeleniumAutomation/Process/ProfileDescriptionProcess.cs

```diff
@@ -0,0 +1,116 @@
+﻿using AventStack.ExtentReports;
+using Mars_SeleniumAutomation.Pages.Components.ProfilePageOverview;
+using Mars_SeleniumAutomation.TestModel;
+using Mars_SeleniumAutomation.Utilities;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mars_SeleniumAutomation.Process
+{
+    public class ProfileDescriptionProcess : Base
+    {
+#pragma warning disable
+        ProfileDescription profileDescriptionObj;
+        Jsonhelper jsonhelperObj;
+        Base baseObj;
+        ExtentTest testreport;
+        private ExtentReports extent;
+        public ProfileDescriptionProcess()
+        {
+            profileDescriptionObj = new ProfileDescription();
+            baseObj = new Base();
+            jsonhelperObj = new Jsonhelper();
+        }
+        public void addDescription()
+        {
+            baseObj.InitializeExtentReports();
+            string testName = "ProfileDescriptiondata";
+            baseObj.SetupTest(testName);
+            ProfileDescriptionTestModel model = new ProfileDescriptionTestModel();
+            List<ProfileDescriptionTestModel> ProfileDescriptiondata = Jsonhelper.ReadTestDataFromJson<ProfileDescriptionTestModel>("D:\\IC Course\\AdvancedNUnit\\Mars-AdvancedTaskNUnit\\Mars-SeleniumAutomation\\JsonData\\ProfileDescriptiondata.json");
+            foreach (var data in ProfileDescriptiondata)
+            {
+                string text = data.textArea;
+                Console.WriteLine(text);
+                string screenshotName = "DescriptionData";
+                baseObj.CaptureScreenshot(screenshotName);
+                profileDescriptionObj.addAndUpdateDescriptionDetails(text);
+                string addedDescription = profileDescriptionObj.getVerifyAddedDescription();
+                Assert.AreEqual(addedDescription, text, "Actual text and expected text do not match");
+                if (testreport != null)
+                {
+                    testreport.Log(Status.Pass, "Test Passed");
+                }
+            }
+        }
+        public void addNegativeDescription()
+        {
+            baseObj.InitializeExtentReports();
```

Comment:
you should move the InitializeExtentReports in a common please do this every where else! you should only initialise the report once and then add tests to the report